src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -139,6 +139,26 @@
       help = "Whether to upload locally executed action results to the remote cache.")
   public boolean remoteUploadLocalResults;
 
+  @Option(
+      name = "incompatible_remote_results_disk_only",
```

Comment:
Can we rename to `incompatible_remote_results_flags_ignore_disk`? The current name makes me think this flag makes remote results flags apply to the disk cache only (which is the opposite of what they do)

Reply:
Sounds good. I prefer just `incompatible_remote_results_ignore_disk` to make it a bit shorter.